openmc/deplete/results.py

```diff
@@ -9,6 +9,7 @@
 import h5py
 import numpy as np
 
+import openmc
```

Comment:
Maybe just import what you need? Not that it matters very much. So you'd instead just do `import openmc.material; import openmc.geometry`, I think.